layouts/partials/footer.html

```diff
@@ -8,7 +8,7 @@
   <div class="container">
     <div class="columns is-vcentered is-variable is-8">
       <div class="column is-one-third">
-        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo">
+        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo" alt="Logo">
```

Comment:
```suggestion
        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo" alt="Keda logo">
```

Reply:
there are 2 alt tags here, shouldn't i add the keda logo one and remove the other

Reply:
Yes, thanks